src/popup/views/GrantAccess/index.tsx

```diff
@@ -1,8 +1,43 @@
 import React from "react";
+import styled from "styled-components";
 import { useLocation } from "react-router-dom";
+
 import { rejectAccess, grantAccess } from "api/internal";
 
-const GrantAccess = () => {
+import { COLOR_PALETTE, FONT_WEIGHT } from "popup/styles";
+import { Button } from "popup/basics";
+
+const GrantAccessEl = styled.div`
+  padding: 2.25rem 2.5rem;
+`;
+const H1 = styled.h1`
+  color: ${COLOR_PALETTE.primary}};
+  font-weight: ${FONT_WEIGHT.light};
+  margin: 1rem 0 0.75rem;
+`;
+const H3 = styled.h3`
+  font-weight: ${FONT_WEIGHT.bold};
+  font-size: 0.95rem;
+  letter-spacing: 0.1px;
+  color: ${COLOR_PALETTE.primary}};
+`;
+const Text = styled.p`
+  font-size: 1.15rem;
+  text-align: center;
+  line-height: 1.9;
+  padding: 1.7rem 2rem;
+  margin: 0;
+`;
+const ButtonContainerEl = styled.div`
+  display: flex;
+  justify-content: space-around;
+  padding: 3rem 1.25rem;
+`;
+const RejectButton = styled(Button)`
```

Comment:
There is a `prop` for `size`. **But** for colors, I didn't add props for `<Button/>` so here I just extended an existing style `Button` and created `RejectButton`.